server/api.js

```diff
@@ -0,0 +1,409 @@
+/* Conventions for this API:
+ *  - GET method for read actions
+ *  - POST method for create actions (think POSTing a new item to a directory)
+ *  - PUT method for update actions (think PUTting an item over an existing item)
+ *  - DELETE method for delete actions
+ *
+ *  - Must authorize using the Authorization header in all requests
+ *    - Presently, this is done as "Bearer (Google ID Token)" as requests should originate from PA
+ *
+ *  - Must return a JSON object containing the following keys:
+ *    - status:   which will be either "success" or "error"
+ *    - data:     the desired JSON object, if there is one
+ *    - message:  a human-readable explanation of the error, if there was one and this is appropriate. Be careful
+ *                to not include anything that will give an attacker extra information
+ */
+
+// Third-party dependencies
+const Validator = require('express-validator')
+
+// In-house dependencies
+const { helpers, googleHelpers } = require('brave-alert-lib')
+const db = require('./db/db')
+
+// authorize function - using Google ID Tokens
+// this should be significantly reworked if providing a limited API to clients
+// NOTE: a route's validation should PRECEED the authorize function, and a route's handler should PROCEED the authorize function;
+//   e.g.: app.method('/api/thing', api.validateThing, api.authorize, api.handleThing)
+async function authorize(req, res, next) {
```

Comment:
Rework this to NOT use Google to authenticate. Maybe just use the Brave API key?

Reason: these API routes may not be implemented from PA, if for a client facing dashboard